gtsam/gtsam.i

```diff
@@ -448,6 +448,16 @@ class Point3 {
   void serialize() const;
 };
 
+#include <gtsam/geometry/Point3.h>
+class Point3Pairs
+{
```

Comment:
Formatting? We use the Google/1TBS style rather than the K&R/Linux kernel style.

Reply:
done, thanks. I moved the `{` brace on the same line as `class Point3Pairs` now